src/index.ts

```diff
@@ -11,4 +11,6 @@ export {
 export {
   WithinContentContext as _SECRET_INTERNAL_WITHIN_CONTENT_CONTEXT,
 } from './utilities/within-content-context';
+
+export {buildColors as UNSTABLE_BuildColors} from './utilities/theme/utils';
```

Comment:
```suggestion
export {buildColors as UNSTABLE_BuildColors} from './utilities/theme';
```

and add an export from `theme/index.ts`, this allows us to quickly look at the local index file to have an idea of what we expose publicly